apis/standard.oam.dev/v1alpha1/rollout_plan_types.go

```diff
@@ -88,7 +87,10 @@ type RolloutPlan struct {
 	NumBatches *int32 `json:"numBatches,omitempty"`
 
 	// The exact distribution among batches.
-	// mutually exclusive to NumBatches
+	// mutually exclusive to NumBatches.
```

Comment:
please mark a TODO here for webhook check.

Reply:
I have already put the check there in this PR (I changed this comment while I am writing the check)